package-res/resources/web/pentaho/type/Instance.js

```diff
@@ -115,6 +116,58 @@ define([
     set type(config) {
       // Class.implement essentially just calls Class#extend.
       if(config) this.type.extend(config);
+    },
+    //endregion
+
+    //region serialization
+    /**
+     * Creates a top-level specification that describes this instance.
+     *
+     * This method creates a new {@link pentaho.type.SpecificationScope} for describing
+     * this instance, and any other instances and types it references,
+     * and then delegates the actual work to {@link pentaho.type.Instance#toSpecInScope}.
+     *
```

Comment:
Don't see why this would be wrong: "This method **creates** ... and then *delegates*". Is it a matter of style?